docs/rules-development-guide-mtr/master.adoc

```diff
@@ -8,11 +8,13 @@ include::topics/templates/document-attributes.adoc[]
 :context: rules-development-guide-mtr
 :rules-development-guide-mtr:
 :_content-type: ASSEMBLY
-= {RulesDevBookName}
+[id="rules-development-guide"]
+= Rules Development Guide
```

Comment:
@raoradhika - this was the change i made to fix the build error coming from Sasha's changes